blockchain/internal/core/transaction.go

```diff
@@ -29,3 +40,67 @@ type Transaction struct {
 	// Data holds additional information related to the transaction.
 	Data string
 }
+
+// Pending transactions (in memory)
+var pendingTransactions []Transaction
+
+// CreateTransaction creates a new transaction and adds it to the pending list
+func CreateTransaction(txType TransactionType, data string) Transaction {
+	tx := Transaction{
+		Header: TransactionHeader{
+			Index:     len(pendingTransactions), // Adequate as long as the blockchain stays in RAM
+			Timestamp: time.Now(),
```

Comment:
We should use `string` as data type, instead of `time.Time`. We used `string` for the Timestamp in the [Block structure](https://github.com/THI-CSI/decentralized_iam_battery_data/blob/65fdedac5f4f7bd55b2b95c196c26d8c9d0dde44/blockchain/internal/core/block.go#L18) as well.